src/Mod/AddonManager/addonmanager_python_deps_gui.py

```diff
@@ -441,21 +461,21 @@ def _current_python_version_is_new(cls) -> bool:
             return True
         return False
 
-    def _load_old_package_list(self) -> List[str]:
+    def _load_old_package_list(self) -> KeysView[str]:
         """Gets the list of packages from the package installation manifest"""
```

Comment:
There's now a mismatch between docstring+function name and return type.

I don't think it makes sense to return a `KeysView[str]`.
We should either:
* change the return to `return list(packages.keys())`, or
* refactor the function, including name docstring, to return an `Iterable[str]`. <-- I would prefer that